ctci/chapter1/1.7.py

```diff
@@ -0,0 +1,34 @@
+#1.7
+#Rotate Matrix: Given an image represented by an NxN matrix, 
+#where each pixel in the image is 4 bytes, write a method to 
+#rotate the image by 90 degrees. can you do this in place?
+from typing import List
+
+#question was for bytes, but we can ignore that :)
+def rotateMatrix(m: List[List[bytes]]) -> List[List[bytes]]:
+    n = len(m)//2
```

Comment:
Given you do range(n) is it necessary for you to floor this value?

Reply:
I think range complained if i gave it a float/none floored value.

Reply:
fair enough.